### ISSUE TITLE
back to the link origin

### ISSUE THREAD

**Issue by 532910** [gsantner/markor#705]
Pressing back on a file that was opened by a link from another file will return to file list, rather than to that file.

**Comment by gsantner** [comment]
@532910 

* Fill out the issue template.
* Which back button? Top left? system?

**Comment by 532910** [comment]
system

**Comment by 532910** [comment]
Markor: 2.1.2 from F-Droid, lineageos 16

**Comment by 532910** [comment]
There is another issue that looks as this one:
1. Open markor, it will show predefined `notes` folder.
2. Go to subfolder.
3. Open some file in this subfolder.
4. Press back button (system or top left, it doesn't matter)
You'll see the the `notes` folder instead of subfolder.

**Comment by gsantner** [comment]
OK thanks - so the behaviour on the top button is wanted like this (hence asked).

**Comment by 532910** [comment]
Really I was wrong: it matters which back button is:
System back button works as expected. Top left back button moves you to `notes` folder.

**Comment by Zoo-M0** [comment]
Could I help you on that? lmk

**Comment by gsantner** [comment]
@Zoo-M0  
Yes. Though it's a bit more complicated and require more effort to get started.  
I saw you are interested in contributing generally (which is very cool btw, Markor very much needs people like you!) - I suggest you to first make familiar with the project, the project hierarchy & behaviour.
